BuildingBlocks/src/BuildingBlocks.API/Mvc/ExceptionFilters/CustomExceptionFilter.cs

```diff
@@ -94,13 +94,10 @@ private HttpError CreateHttpErrorApplicationException(ApplicationException appli
     {
         if (applicationException is QuotaExhaustedException quotaExhautedException)
         {
-            return quotaExhautedException.ExhaustedMetricStatuses.Select(it => 
-                new Dictionary<string, object>
-                {
-                    { "MetricKey", it.MetricKey },
-                    { "IsExhaustedUntil", it.IsExhaustedUntil }
-                }
-            );
+            return quotaExhautedException.ExhaustedMetricStatuses.Select(m => new {
+                m.MetricKey,
+                m.IsExhaustedUntil
```

Comment:
Not good. This results in the same problem we would have when returning the exception data directly: when we change the property name in the exception, the returned JSON would automatically change as well, leading to a change in the API.